README.md

```diff
@@ -73,7 +87,7 @@ Frontend URL: https://xxxxxxxxx.cloudfront.net
 The sign-up screen will appear as shown above, where you can register your email and log in.
 
 > [!Important]
-> This deployment method allows anyone with the URL to sign up. For production use, we strongly recommend adding IP address restrictions or disabling self-signup to mitigate security risks. To set up, [Deploy using CDK](#deploy-using-cdk) for IP address restrictions or [Disable self sign up](#disable-self-sign-up).
+> This deployment method allows anyone with the URL to sign up if optional parameters are not configured. For production use, we strongly recommend adding IP address restrictions and disabling self-signup to mitigate security risks (Defining the allowed-signup-email-domains to allow only your emails from your comapny domain to be able to sing-up to restrict the users). For ip address restriction use both ipv4-ranges and ipv6-ranges and to disable self-signup use disable-self-register when executing ./bin
```

Comment:
>when executing ./bin

Please enclose ./bin with quoations like `./bin.sh`

Reply:
>use both ipv4-ranges and ipv6-ranges

Is it necessary to configure both IPv4 and IPv6 ranges? Depending on the environment, I think setting only one of them might be sufficient.

Reply:
If you set just one, the other will allow all.

Example, if you set ipv4 to 192.0.2.0/24, IPv6 will have by default all ranges enabled so you would only be restricting ipv4 and not v6 as well.

Reply:
Agree, but since not all clients have IPv6 addresses, it's best to recommend configuring access controls using either IPv4 or IPv6 rather than enforcing it in the code. But current notation also can be understood that the user does not need to specify both of them, so let me resolve.